server/nodejs/controllers/AliasesService.js

```diff
@@ -187,11 +187,35 @@ exports.aliasupdatewhitelist = function(args, res, next) {
   syscoinClient.aliasUpdateWhitelist.apply(syscoinClient, arr);
 }
 
-exports.syscointxfund = methodGenerator.generateGenericSyscoinMethod([
-  { prop: 'hexstring' },
-  { prop: 'addresses' }
-], syscoinClient.syscoinTxFund, 'syscointxfund', 'POST');
+// exports.syscointxfund = methodGenerator.generateGenericSyscoinMethod([
+//   { prop: 'hexstring' },
+//   { prop: 'addresses' }
+// ], syscoinClient.syscoinTxFund, 'syscointxfund', 'POST');
 
+exports.syscointxfund = function(args, res, next) {
+  var argList = [
+    { prop: 'hexstring' },
+    { prop: 'addresses' }
+  ];
+
+  var cb = function(err, result, resHeaders) {
+    res.setHeader('Content-Type', 'application/json');
+
+    if (err) {
+      return commonUtils.reportError(res, err);
+    }
+
+    commonUtils.log('Syscoin TX fund:', result, "syscointxfund");
+    res.end(JSON.stringify(result));
+  };
+ // Convert the Addresses array to string
+  var actualAddresses = args.request.value['addresses']
```

Comment:
Can we ensure that request, request.value, and request.value['addresses'] exist?